### ISSUE TITLE
[FEATURE] More searching capabilities for the Cases page

### ISSUE THREAD

**Issue by rediasec** [reflexsoar/reflex-api#257]
**Is your feature request related to a problem? Please describe.**
I have encountered several situations where I wanted to look up a case but could not remember its title, only some observables or details in the description.

**Describe the solution you'd like**
A `Search` box that will search all the fields in each Case for the entered strings ...?

**Describe alternatives you've considered**
Using the Title search with several strings and the other available filters until I found the case wanted.

**Additional context**
![image](https://user-images.githubusercontent.com/61625853/191255779-34e11824-6686-4d4d-af1e-30083c569c27.png)

**Sponsored Feature**
Unfortunately, not. Just a suggestion from a user who uses the tool in his daily work.

**Comment by n3tsurge** [comment]
I would agree this is a much needed thing, will try to get it in to this next release if not the one after that.

**Comment by n3tsurge** [comment]
In the next release you will have the ability to filter on Title, Description, Observables and Comments

![image](https://user-images.githubusercontent.com/1574741/191401674-f3dd255c-6ce0-4f5f-925e-5e0e32bbfcba.png)

**Comment by rediasec** [comment]
This is exactly what I needed. Looking forward to the next release! :partying_face:  _(I saw that there are even more new and exciting features)_

**Comment by n3tsurge** [comment]
QA Note: Title is a `keyword` and not `text` so wildcard searches are required and are case sensitive.  Adding a `text` field to `title` to allow for case insensitive searches and partial matching
